lms/djangoapps/commerce/urls.py

```diff
@@ -4,10 +4,10 @@
 
 from django.conf.urls import patterns, url
 
-from .views import OrdersView, checkout_cancel
+from .views import BasketsView, checkout_cancel
 
 urlpatterns = patterns(
     '',
-    url(r'^orders/$', OrdersView.as_view(), name="orders"),
+    url(r'^baskets/$', BasketsView.as_view(), name="baskets"),
```

Comment:
Is there a risk that clients might still try to contact the old end-point if they're using cached JS of the logistration page?  It might make sense to leave the old end-point for an initial release, then remove it in the following release.